### ISSUE TITLE
homebrew package

### ISSUE THREAD

**Issue by soonhokong** [dreal/dreal3#161]
Issue: the latest version of apple clang (`clang-700.0.72`) still does not support `thread_local` yet, as of 2015/09/20. For now, as a temporary workaround, let's remove `thread_local` since we don't need it until we implement parallelism.

In the future, we need to rely on [Boost's thread local storage](http://www.boost.org/doc/libs/1_35_0/doc/html/thread/thread_local_storage.html) for OSX if Apple doesn't support `thread_local`.

Note: here is a discussion why Apple doesn't support the feature - http://stackoverflow.com/a/29929949

**Comment by soonhokong** [comment]
there is some problem in compiling Coin-Clp:

```
Undefined symbols for architecture x86_64:
  "std::terminate()", referenced from:
      ___clang_call_terminate in libCoinUtils.3.10.6.dylib-master.o
```

**Comment by soonhokong** [comment]
This problem got fixed by upgrading Coin-Clp from 1.16.6 to 1.16.7: https://github.com/dreal-deps/clp-1.16/commit/3c7d2716661fe0bab3957f9bf87d3af841ce8528

**Comment by soonhokong** [comment]
The following should work (tested on multiple macs):

```
brew install dreal/dreal/dreal
```

FYI, it only install dReal3 and as a result dReach is using dReal3.
